manifold-endpoint/README.md

```diff
@@ -0,0 +1,83 @@
+# Manifold Platform Endpoint on CentOS
+By clicking "Deploy to Azure" you agree to the Terms and Conditions below.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Ftpinckard%2Fazure-quickstart-templates%2Fmaster%2Fmanifold-endpoint%2Fazuredeploy.json" target="_blank"><img src="http://azuredeploy.net/deploybutton.png"/></a>
+# This template installs the Manifold Platform REST Endpoint on a CentOS VM.
+ 
+#
+
```

Comment:
It would also be good to add a bit more documentation about what Manifold Platform is and documentation links